src/elli_http.erl

```diff
@@ -745,7 +745,7 @@ split_path(Path) ->
           P =/= <<>>].
 
 %% @doc Split the URL arguments into a proplist.
-%% Lifted from `cowboy_http:x_www_form_urlencoded/2'.
+%% Lifted from cowboy_http:x_www_form_urlencoded/2.
```

Comment:
Is there some particular reason to drop the `` ` `` and `` ' `` here?

Reply:
No special reason. Lemme re-check the links.

Reply:
Most likely because it's a link to an external project, so it wouldn't know where to point, but I can check...

Reply:
Unless you mean you want to keep the notation even if there's no link, which seems to be the case.